cmd/bundle-extractor/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/pkg/sqlite"
+)
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:   "bundle-extractor",
+		Short: "bundle-extractor",
+		Long:  "Top level CLI for bundle operations",
+	}
+
+	serveCmd := &cobra.Command{
+		Use:   "extract",
+		Short: "Extracts the data in a bundle directory via ConfigMap",
+		Long:  "Extract takes as input a directory containing manifests and writes the per file contents to a ConfipMap",
+
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if debug, _ := cmd.Flags().GetBool("debug"); debug {
+				logrus.SetLevel(logrus.DebugLevel)
+			}
+			return nil
+		},
+
+		RunE: runServeCmd,
+	}
+	serveCmd.Flags().Bool("debug", false, "enable debug logging")
+	serveCmd.Flags().StringP("kubeconfig", "k", "", "absolute path to kubeconfig file")
+	serveCmd.Flags().StringP("manifestsdir", "m", "/", "path to directory containing manifests")
+	serveCmd.Flags().StringP("configmapname", "c", "", "name of configmap to write bundle data")
+	serveCmd.Flags().StringP("namespace", "n", "openshift-operator-lifecycle-manager", "namespace to write configmap data")
+	serveCmd.MarkPersistentFlagRequired("configmapname")
+	rootCmd.AddCommand(serveCmd)
+
+	if err := rootCmd.Execute(); err != nil {
+		logrus.Error(err)
+	}
+}
+
+func runServeCmd(cmd *cobra.Command, args []string) error {
+	manifestsDir, err := cmd.Flags().GetString("manifestsdir")
+	if err != nil {
+		return err
+	}
+	kubeconfig, err := cmd.Flags().GetString("kubeconfig")
+	if err != nil {
+		return err
+	}
+	configmapName, err := cmd.Flags().GetString("configmapname")
+	if err != nil {
+		return err
+	}
+	namespace, err := cmd.Flags().GetString("namespace")
+	if err != nil {
+		return err
+	}
+
+	loader := sqlite.NewConfigMapLoaderForDirectory(configmapName, namespace, manifestsDir, kubeconfig)
```

Comment:
could we find a better place to put this than in the `sqlite` package?

Reply:
I figured this would be brought up. But I didn't know where else to put it. Suggestions?

Reply:
Moved to new configmap package.